go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/ElrondNetwork/elastic-indexer-go
 go 1.17
 
 require (
-	github.com/ElrondNetwork/elrond-go-core v1.1.24
-	github.com/ElrondNetwork/elrond-go-logger v1.0.8
-	github.com/ElrondNetwork/elrond-vm-common v1.3.26
+	github.com/ElrondNetwork/elrond-go-core v1.1.25-0.20221114131354-7a10b0122311
```

Comment:
when the PRs are ready on the `elrond-go-core` repository and `elrond-vm-common` please add proper tags here.

Reply:
:+1: